### ISSUE TITLE
Bike sign 🚲 preventens to show images in edit mode

### ISSUE THREAD

**Issue by jspille** [tamlok/vnote#468]
OS Version (操作系统): Windows

VNote Version (VNote版本): Latest (最新) 2.0

Symptoms (表现):

In Edit-Mode I can see image previews - that is the reason why I use vnode!
Images following this Unicode <🚲> sign in a text are not shown in edit-mode, only in view-mode.

Example:

```
![x](link)  <== image are shown
A Text with 🚲 in it
![x](link)  <== image *not* shown
```

How to Repro (如何复现):

Use the example above

**Comment by tamlok** [comment]
Already fixed. Will be included in next release.

**Comment by tamlok** [comment]
Fixed in 2.1. Please try it out. Thanks!
